### PR TITLE
[Fix] Fruit Description Fix

### LINKED ISSUE #1065: Fruit naming
In grown.dm, lines 90 to 116, there are references to trekchem chemicals for fruit descriptions, IE pacid for acidic description. This'll need updating to goofchem for proper descriptioning.

### PR DESCRIPTION
**Pull request by FalseIncarnate**
Fixes checks for old chems to use ones for new chems. Some of these
chemicals currently aren't in any fruits (like juices) but may appear
once we expand the chem list for random seeds.

Will update the descriptions to account for new chems at such time that
goof's PR is finished if he doesn't include it.

Fixes #1065

### CODE REVIEW

File: code/modules/hydroponics/grown.dm [hunk 1]
```diff
@@ -103,17 +103,17 @@
 			descriptors |= "sweet-sour"
 		if(reagents.has_reagent("radium") || reagents.has_reagent("uranium"))
 			descriptors |= "radioactive"
-		if(reagents.has_reagent("amatoxin") || reagents.has_reagent("toxin"))
+		if(reagents.has_reagent("amanitin") || reagents.has_reagent("toxin"))
 			descriptors |= "poisonous"
-		if(reagents.has_reagent("psilocybin") || reagents.has_reagent("space_drugs"))
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/grown.dm]: this reagent still exists.

Comment (FalseIncarnate) [inline code/modules/hydroponics/grown.dm]: Indeed it does, probably should change that to check for lsd, psilo, or drugs

**Comment by FalseIncarnate** [comment]
Re-added psilocybin to the descriptor check, in addition to space drugs and LSD. These are all hallucinogens, so they all will trigger the hallucinogenic descriptor.

File: code/modules/hydroponics/grown.dm [hunk 2]
```diff
@@ -103,17 +103,17 @@
...
+		if(reagents.has_reagent("lsd") || reagents.has_reagent("space_drugs") || reagents.has_reagent("psilocybin"))
 			descriptors |= "hallucinogenic"
-		if(reagents.has_reagent("bicaridine"))
+		if(reagents.has_reagent("styptic_powder"))
 			descriptors |= "medicinal"
```
Comment (Iamgoofball) [inline code/modules/hydroponics/grown.dm]: should be powdery or rough

**Pull request merged**